Userland/Libraries/LibWeb/Geometry/DOMMatrixReadOnly.cpp

```diff
@@ -262,6 +266,7 @@ JS::NonnullGCPtr<DOMPoint> DOMMatrixReadOnly::transform_point(DOMPointReadOnly c
     Vector4<double> point_vector { point.x(), point.y(), point.z(), point.w() };
 
     // 6. Set pointVector to pointVector pre-multiplied by matrix.
+    // This is really a post multiply because of the transposed m_matrix.
```

Comment:
This commit should happen before the introduction of this vector-matrix multiplication, since otherwise the associated tests will fail.

Reply:
I have moved that comment and the old `m_matrix.transpose()` back to the first commit, so that the tests on that commit still work :)